kitsune/community/templates/community/email/first_l10n.html

```diff
@@ -0,0 +1,79 @@
+{# this does not need translated, since it is being sent to localizers #}
+{% extends 'email/base.html' %}
+
+{% block content %}
+    <p>
+        Thank you for your first contribution to our Knowledge Base in your
+        language! You can consider yourself an Open Web rockstar on the rise
+        :-) Your contribution will now undergo review by our awesome community
```

Comment:
This sentence doesn't have an ending punctuation. Maybe it should be:

"Open Web rockstar on the rise :-) ."

or

"Open Web rockstar on the rise. :-)"

?